MantleTests/MTLModelDifferenceSpec.m

```diff
@@ -10,10 +10,32 @@
 
 QuickSpecBegin(MTLModelDifferenceSpec)
 
-it(@"should raise on mismatching argument type", ^{
+it(@"should return message about expected type", ^{
   MTLTestModel *model = [[MTLTestModel alloc] init];
   NSAssert(model, @"model must be non nil");
-  expect([model differenceFrom:@1]).to(raiseException());
+  MTLTestModel *nilModel = nil;
+  NSArray *difference = [model differenceFrom:nilModel];
```

Comment:
Just pass `differenceFrom:nil`

Reply:
No can do. Xcode complains I'm passing `nil` to a method with non-nil argument.